crates/client/data-availability/src/lib.rs

```diff
@@ -76,57 +82,82 @@ where
 {
     pub async fn prove_current_block(da_mode: DaMode, client: Arc<C>, madara_backend: Arc<mc_db::Backend<B>>) {
         let mut storage_event_st = client
-            .storage_changes_notification_stream(None, None)
+            .storage_changes_notification_stream(None, Some(&[(SubStorageKey(twox_128(PALLET_STARKNET).into()), None)]))
```

Comment:
I think you can go `(None, None)` unti we get some light on https://github.com/paritytech/polkadot-sdk/issues/1989